src/SFA.DAS.EmployerIncentives.Api/Types/UpdateIncentiveApplicationRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Commands;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerIncentives.Api.Types
+{
+    public class UpdateIncentiveApplicationRequest
+    {
+        public Guid IncentiveApplicationId { get; set; }
+        public long AccountId { get; set; }
```

Comment:
Do we need accountid and accountlegalentityId on the request?

Reply:
well, in theory we don't need either of them. added for consistency / future checks?

Reply:
YAGNI !

Reply:
YAGNI it is then 😄 -- REMOVED ❌